src/construction.cpp

```diff
@@ -198,13 +198,13 @@ void construction_menu()
     const int w_y0 = ( TERMY > w_height ) ? ( TERMY - w_height ) / 2 : 0;
     const int w_x0 = ( TERMX > w_width ) ? ( TERMX - w_width ) / 2 : 0;
     catacurses::WINDOW_PTR w_con_ptr {catacurses::newwin( w_height, w_width, w_y0, w_x0 )};
-    WINDOW *const w_con = w_con_ptr.get();
+    catacurses::window const w_con = w_con_ptr.get();
 
     const int w_list_width = int( .375 * w_width );
     const int w_list_height = w_height - 4;
     const int w_list_x0 = 1;
     catacurses::WINDOW_PTR w_list_ptr {catacurses::newwin( w_list_height, w_list_width, w_y0 + 3, w_x0 + w_list_x0 )};
-    WINDOW *const w_list = w_list_ptr.get();
+    catacurses::window const w_list = w_list_ptr.get();
```

Comment:
`const` has changed its meaning after applying find-and-replace. It's now related to the `w_list` variable itself, not the referenced object (since `w_list` is not a pointer anymore). This implies the more idiomatic order: `const catacurses::window ...`

Reply:
It hasn't changed its meaning. It was already related to the variable itself. `w_list` was a const pointer to a (variable) `WINDOW`.

I generally refrain from changing this kind of thing as it just creates clutter in the commit log and has *no* effect. And because we don't have a style rule for this, the next coder may change it back anyway.